### PR TITLE
Add sprint planning and definition of done to manual

### PR DESCRIPTION
**Pull request by suzannehamilton**
Summarise the discussion we had as a team about how we plan cards and how we know when they're "done".

These are intended as useful checklists to remind us of important things to check rather than rigid definitions. They should help us rather than getting in the way, and we should update them or manage things differently if they don't help in practice.

### CODE REVIEW

File: manual/development-process/plan-story.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+We should consider whether any new components or interactions require any
+accessibility design (such as [ARIA roles][aria]) or testing once the feature
+is complete.
+
+[aria]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles
+
+### Security considerations
+
+We should think about whether a new feature could potentially introducing
```
Comment (TomJKing) [inline manual/development-process/plan-story.md:126]: Should read: "... whether a new feature could potentially **introduce** security ..."

File: manual/development-process/close-card.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+* any relevant documentation has been written or updated
+
+## Tasks
+
+A task is "done" if:
+
+* the code has been reviewed by another developer
+* the code is merged to the "master" branch
+* the code has been deployed to production (or staging, if we don't have a
+  production environment yet)
```
Comment (TomJKing) [inline manual/development-process/close-card.md:33]: Maybe reword: "... (or staging, if we don't have a production environment **available)**"

File: manual/development-process/close-card.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+with a good understanding of the component being changed. Developers should use
+their judgement to decide whether a task requires a more experienced reviewer.
+
+## Stories
+
+A story is "done" if:
+
+* all code has been reviewed by another developer
+* all the subtasks are merged to the "master" branch
+* all the subtasks have been deployed to production (or staging, if we don't
```
Comment (TomJKing) [inline manual/development-process/close-card.md:17]: Maybe reword: "... (or staging, if we don't have a production environment **available**)"

File: manual/development-process/plan-story.md (hunk 1 above)
Comment (suzannehamilton) [reply manual/development-process/plan-story.md]: Fixed, thanks.

File: manual/development-process/close-card.md (hunk 2 above)
Comment (suzannehamilton) [reply manual/development-process/close-card.md]: Reworded. Hopefully we can remove this caveat altogether soon!

File: manual/development-process/close-card.md (hunk 3 above)
Comment (suzannehamilton) [reply manual/development-process/close-card.md:17]: Reworded.
